packages/types/src/constants.ts

```diff
@@ -126,3 +126,7 @@ export const PACKAGES_DESCRIPTIONS: Record<AllPackages, string> = {
   '@storybook/testing-react': 'Testing utilities that allow you to reuse your stories in your unit tests',
   '@storybook/testing-vue3': 'Testing utilities that allow you to reuse your stories in your unit tests',
 } as const
+
+export const DEFAULT_E2E_SPEC_PATTERN = 'cypress/e2e/**/*.cy.{js,jsx,ts,tsx}'
```

Comment:
are we not able to pull this value from packages/config?

Reply:
Good idea, I'll grab them from the options. Less duplication that way